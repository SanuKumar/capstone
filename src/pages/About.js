import React from 'react'
import { Card, Container, Row, Col, Carousel } from 'react-bootstrap'

const About = () => {

  const imageList = [
    "https://www.smartertech.id/wp-content/uploads/2021/08/keeppack-1.png",
    "https://static.vecteezy.com/system/resources/previews/005/441/345/non_2x/inventory-management-with-goods-demand-and-stock-supply-tiny-person-concept-vector.jpg",
    "https://miro.medium.com/max/1000/1*ka3E4I_7AFezzqPH0K3Ncw.jpeg",
    "https://safsms.com/blog/wp-content/uploads/2017/06/what-is-inventory-management.png"
  ]

  return (
    <>
      <h2 style={{ textAlign: "center", margin: "-4rem 0 2rem 0" }}>About Product Inventory</h2>
      <Container>
        <hr />
        <Row>
          <Col xs={12} sm={6}>
            <Card>
              <Carousel variant="dark" indicators controls>
                {imageList && imageList.map((img) => (
                  <Carousel.Item interval={1000} key={img}>
                    <Card.Img variant="top" height={400} width={200} src={img} alt="about" />
                  </Carousel.Item>
                ))}
              </Carousel>
            </Card></Col>
          <Col xs={12} sm={6}>
            <div style={{ fontSize: "25px", textAlign: "justify" }}>
              Product inventory is used to record information about the amount of available goods 
              and products our company offers. Start by entering a product's name, inventory ID 
              number, description, manufacture, category, location, price and quantity.
            </div>
            <hr />
            <h4>Features Available</h4>
            <ul>
              <li>Add, Edit and Delete Product available for register user's</li>
              <li>Search and Filter products</li>
              <li>User's Registration</li>
              <li>Top Product Rated Chart</li>
            </ul>
          </Col>
        </Row>

      </Container>
    </>
  )
}

export default About