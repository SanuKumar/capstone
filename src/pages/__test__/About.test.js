import React from "react"
import { render, cleanup } from "@testing-library/react"
import About from "../About"

afterEach(cleanup)

describe("About page snapshot", () => {
  test("renders correctly", () => {
    const { asFragment } = render(<About />);
    expect(asFragment()).toMatchSnapshot();
  })
})