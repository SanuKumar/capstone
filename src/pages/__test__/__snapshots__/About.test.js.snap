// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page snapshot renders correctly 1`] = `
<DocumentFragment>
  <h2
    style="text-align: center; margin: -4rem 0px 2rem 0px;"
  >
    About Product Inventory
  </h2>
  <div
    class="container"
  >
    <hr />
    <div
      class="row"
    >
      <div
        class="col-sm-6 col-12"
      >
        <div
          class="card"
        >
          <div
            class="carousel slide carousel-dark"
          >
            <div
              class="carousel-indicators"
            >
              <button
                aria-current="true"
                aria-label="Slide 1"
                class="active"
                data-bs-target=""
                type="button"
              />
              <button
                aria-current="false"
                aria-label="Slide 2"
                data-bs-target=""
                type="button"
              />
              <button
                aria-current="false"
                aria-label="Slide 3"
                data-bs-target=""
                type="button"
              />
              <button
                aria-current="false"
                aria-label="Slide 4"
                data-bs-target=""
                type="button"
              />
            </div>
            <div
              class="carousel-inner"
            >
              <div
                class="active carousel-item"
                interval="1000"
              >
                <img
                  alt="about"
                  class="card-img-top"
                  height="400"
                  src="https://www.smartertech.id/wp-content/uploads/2021/08/keeppack-1.png"
                  width="200"
                />
              </div>
              <div
                class="carousel-item"
                interval="1000"
              >
                <img
                  alt="about"
                  class="card-img-top"
                  height="400"
                  src="https://static.vecteezy.com/system/resources/previews/005/441/345/non_2x/inventory-management-with-goods-demand-and-stock-supply-tiny-person-concept-vector.jpg"
                  width="200"
                />
              </div>
              <div
                class="carousel-item"
                interval="1000"
              >
                <img
                  alt="about"
                  class="card-img-top"
                  height="400"
                  src="https://miro.medium.com/max/1000/1*ka3E4I_7AFezzqPH0K3Ncw.jpeg"
                  width="200"
                />
              </div>
              <div
                class="carousel-item"
                interval="1000"
              >
                <img
                  alt="about"
                  class="card-img-top"
                  height="400"
                  src="https://safsms.com/blog/wp-content/uploads/2017/06/what-is-inventory-management.png"
                  width="200"
                />
              </div>
            </div>
            <a
              class="carousel-control-prev"
              href="#"
              role="button"
              tabindex="0"
            >
              <span
                aria-hidden="true"
                class="carousel-control-prev-icon"
              />
              <span
                class="visually-hidden"
              >
                Previous
              </span>
            </a>
            <a
              class="carousel-control-next"
              href="#"
              role="button"
              tabindex="0"
            >
              <span
                aria-hidden="true"
                class="carousel-control-next-icon"
              />
              <span
                class="visually-hidden"
              >
                Next
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="col-sm-6 col-12"
      >
        <div
          style="font-size: 25px; text-align: justify;"
        >
          Product inventory is used to record information about the amount of available goods and products our company offers. Start by entering a product's name, inventory ID number, description, manufacture, category, location, price and quantity.
        </div>
        <hr />
        <h4>
          Features Available
        </h4>
        <ul>
          <li>
            Add, Edit and Delete Product
          </li>
          <li>
            Multiple Products Deletion
          </li>
          <li>
            Search and Filter products
          </li>
          <li>
            User's Registration
          </li>
          <li>
            Top Product Rated Chart
          </li>
        </ul>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
